network-api/networkapi/buyersguide/views.py

```diff
@@ -100,6 +100,7 @@ def product_view(request, slug):
         'product': product.to_dict(),
         'mediaUrl': MEDIA_URL,
         'coralTalkServerUrl': settings.CORAL_TALK_SERVER_URL,
+        'pageTitle': f'*privacy not included - {product.name}',
```

Comment:
I don't know Django enough - do we need to check whether `product` or `product.name` exists?

Reply:
@Pomax ^

Reply:
coooooool